### DISCUSSION TITLE
Can not unmap keybinding

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [brookhong/Surfingkeys#2182] [Q&A] [answered]
### Setup
- Firefox 131.0.3, SK 1.16.1
- Vivaldi 6.9.3447.54, SK 1.17.3
- OS Win 10 pro 22H2

### Edit
It seems this issue is reported at #2087. And the problem is how the sample config is wrong.
The fix is adding missing `api` before `Front` and use `<Ctrl-y>` instead of `<ctrl-y>`.
```js
// an example to create a new mapping `ctrl-y`
api.mapkey('<Ctrl-y>', 'Show me the money', function() {
    api.Front.showPopup('a well-known phrase uttered by characters in the 1996 film Jerry Maguire (Escape to close).');
});
```

To unmap `<alt-s>`, I guess it needs to be remapped first and use `<Alt-s>`.

The first I would do is to rewrite this sample config to not misleading user for his first experience.

For history 👇

### Issue

Hi,

I just installed this extensions. I opened settings in advanced mode.

The sample is as fallow:
```js
// an example to create a new mapping `ctrl-y`
api.mapkey('<ctrl-y>', 'Show me the money', function() {
    Front.showPopup('a well-known phrase uttered by characters in the 1996 film Jerry Maguire (Escape to close).');
});

// an example to replace `T` with `gt`, click `Default mappings` to see how `T` works.
api.map('gt', 'T');

// an example to remove mapkey `Ctrl-i`
api.unmap('<ctrl-i>');
```
followed by theme.

I added
```js
api.unmap('<alt-s>');
```
because it overrides an existing shortkey I use.
I saved my settings then tested in a new tab. 

### Settings that were applied
- Sample Theme: dark mode
- `api.map('gt', 'T');` added `gt`

### Settings NOT applied

- `<ctrl-i>` and `<alt-s>` are still mapped.
- `<ctrl-y>` does nothing. The help screen doesn’t mention then command.

### What I tried
I saved multiple times. Restarted browsers. Disable/Enable advanced. Tested in fresh profiles.

Same result in 1.16.1 and 1.17.3

Did I miss something ?

**Comment by Fred-Vatin** [comment, marked as answer]
It seems this issue is reported at #2087. And the problem is how the sample config is wrong.
The fix is adding missing `api` before `Front` and use `<Ctrl-y>` instead of `<ctrl-y>`.
```js
// an example to create a new mapping `ctrl-y`
api.mapkey('<Ctrl-y>', 'Show me the money', function() {
    api.Front.showPopup('a well-known phrase uttered by characters in the 1996 film Jerry Maguire (Escape to close).');
});
```

To unmap `<alt-s>`, I guess it needs to be remapped first and use `<Alt-s>`.

The first I would do is to rewrite this sample config to not misleading user for his first experience.

**Reply by pablospe** [reply]
I tried with this but didn't work:
`api.unmap('<Ctrl-i>');`
